src/sdg/api/serializers/producten.py

```diff
@@ -115,18 +165,19 @@ class Meta(LocatieBaseSerializer.Meta):
             "uuid": {
                 "required": False,
                 "read_only": False,
+                "help_text": "De uuid van een specifieke organisatie (https://en.wikipedia.org/wiki/Universally_unique_identifier).",
             },
         }
 
 
 class ProductLokaleOverheidSerializer(serializers.HyperlinkedModelSerializer):
-    """Serializer that exposes a small subset of the fields for an organization, used in references to an organization.
+    """De organisatie die dit product levert en de teksten hiervan beheerd dit geven we aan met een van de volgende velden:
```

Comment:
"De organisatie die dit product levert en de teksten hiervan beheert. Dit geven we aan met een van de volgende velden"